easybuild/framework/extensioneasyblock.py

```diff
@@ -101,6 +102,10 @@ def run(self, unpack_src=False):
             targetdir = os.path.join(self.master.builddir, remove_unwanted_chars(self.name))
             self.ext_dir = extract_file("%s" % self.src, targetdir, extra_options=self.unpack_options)
 
+            if self.cfg['start_dir'] and os.path.isdir(self.cfg['start_dir']):
```

Comment:
use `self.start_dir` rather than `self.cfg['start_dir']` when just reading the value, cfr. #2339 (we should do a cleanup sweep in framework & easyblocks for this)